current/runtime/storage.md

```diff
@@ -56,38 +244,129 @@ decl_storage! {
 }
 ```
 
-## Verify First, Write Last
+### Genesis Config
 
-TODO
+You can define
+[an optional `GenesisConfig`](https://substrate.dev/rustdocs/master/frame_support/macro.decl_storage.html#genesisconfig)
+struct in order to initialize Storage Items in the genesis block of your blockchain.
 
-## Storage Cache
+// TODO
```

Comment:
@thiolliere what info do you think should go here?

Reply:
GenesisConfig is generated by decl_storage macro, it does initialize storage items for the genesis block of the blockchain.

Its logic is defined by:
* `config` and `build` attribute when declaring an individual storage.
* add_extra_genesis informations. which allows to add fields in the GenesisConfig struct and a build function to build storages using those fields.

(I can be more precise actually we execute first for each storage the build closure if it exist, or if no build attribute but a config attribute then we add into storage the value found in config. Then we execute the add_extra_genesis::build function.)